app/src/main/java/io/neurolab/main/NeuroLab.java

```diff
@@ -75,18 +75,14 @@
     private static int baudRate = 9600;
     private static boolean deviceConnected;
     private static String deviceData;
-    private Menu menu;
-    private int launcherSleepTime;
     private static AppUpdateManager appUpdateManager;
     private static Task<AppUpdateInfo> appUpdateInfoTask;
+    private Menu menu;
+    private int launcherSleepTime;
     private UsbSerialInterface.UsbReadCallback readCallback = new UsbSerialInterface.UsbReadCallback() { //Defining a Callback which triggers whenever data is read.
         @Override
         public void onReceivedData(byte[] arg0) {
-            try {
-                deviceData = new String(arg0, "UTF-8");
-            } catch (UnsupportedEncodingException e) {
-                e.printStackTrace();
-            }
+            deviceData = new String(arg0, StandardCharsets.UTF_8);
```

Comment:
Why this change? Please do not change any of the hardware-related code now.

Reply:
That's a pure mistake. I'll make the appropriate changes.